src/pages/accountAndSettings/index.tsx

```diff
@@ -0,0 +1,121 @@
+import { Tab } from "@headlessui/react";
+import { User } from "@prisma/client";
+import { useMutation, useQuery, useQueryClient } from "@tanstack/react-query";
+import axios from "axios";
+import clsx from "clsx";
+import { GetServerSideProps } from "next";
+import { getSession } from "next-auth/react";
+import Head from "next/head";
+import Link from "next/link";
+import { useState } from "react";
+import { FiChevronLeft } from "react-icons/fi";
+import AccountView from "./account";
+import Settings from "./settings";
+
+export const getUserInfo = async (email: string) => {
+	const John: User = await axios
```

Comment:
`return await axios...` instead of using a binding (`const John`) here please

Reply:
okis